File: src/test/java/no/nav/fo/veilarbregistrering/profilering/ProfileringTestdataBuilder.java
package no.nav.fo.veilarbregistrering.profilering;

public class ProfileringTestdataBuilder {
    public static Profilering lagProfilering() {
        return new Profilering()
                .setInnsatsgruppe(Innsatsgruppe.STANDARD_INNSATS)
                .setAlder(62)
                .setJobbetSammenhengendeSeksAvTolvSisteManeder(false);
    }
}

File: src/test/java/no/nav/fo/veilarbregistrering/profilering/ProfileringTestdataBuilder.kt
package no.nav.fo.veilarbregistrering.profilering

object ProfileringTestdataBuilder {
    fun lagProfilering(): Profilering {
        return Profilering()
            .setInnsatsgruppe(Innsatsgruppe.STANDARD_INNSATS)
            .setAlder(62)
            .setJobbetSammenhengendeSeksAvTolvSisteManeder(false)
    }
}
